notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<p>W dalszym ciągu mamy trochę za dużo kategorii szkół. Spróbujmy sprowadzić to do tych konkretnych poziomów: https://pl.wikipedia.org/wiki/Struktura_szkolnictwa_w_Polsce#Etapy_edukacji_w_Polsce_(od_2017)</p><p>Wszystko, co nie jest przedszkolem lub szkołą może trafić do kategorii <code>Other</code>.  Jeżeli dalej zostają nam szkoły, które mają kilka poziomów lub nie jesteśmy w stanie jednoznacznie tego poziomu określić to zróbmy może póki co dodatkową kategorię <code>School_Other</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Tak wiem bo poprawiałam tylko skrypty do budowania datasetu :) zajme sie tym niebawem :)

Reply:
ok